### PR TITLE
1.0.1

### LINKED ISSUE #6: Add field for Google Analytics to global configuration

### LINKED ISSUE #7: Add field for Google Webmasters authentication in global configuration

### PR DESCRIPTION
**Pull request by austinstarin**
Configure Google Analytics and site verification. Close #6 and close #7.

### CODE REVIEW

**Comment by austinstarin** [comment]
We need to arrangeFields on these in a sensible way too.

File: lib/modules/apostrophe-seo-global/index.js [hunk 1]
```diff
@@ -1,4 +1,21 @@
...
   improve: 'apostrophe-global',
-  seo: false
+  seo: false,
+  construct: (self, options) => {
+    const seoGoogleFields = options.seoGoogleFields ? [
+      {
+        name: 'seoGoogleTrackingId',
+        label: 'Google Tracking ID',
+        type: 'string',
+        help: 'Tracking ID provided by Google for Google Analytics.'
```
Comment (abea) [inline lib/modules/apostrophe-seo-global/index.js:10]: I think this should specify it's using the Tag Manager ID. I think the old code version would be different.

Comment (austinstarin) [reply lib/modules/apostrophe-seo-global/index.js:10]: They are using "Tracking ID" in docs, am I missing something?

https://support.google.com/analytics/answer/1008080?hl=en

Comment (abea) [reply lib/modules/apostrophe-seo-global/index.js:10]: If that's consistent with their language then that's cool. I got my head in 2014 GA still.

**Review by bgantick — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -68,3 +69,26 @@ Add the following to `layout.html` that all of your pages extend, or to `outerLa
   {% endif %}
 {% endblock %}
 ```
+
+If you would like to configure additional fields to allow an editor to add a Google Analytics tracking ID and a Google site verification ID you can do so by setting `seoGoogleFields: true` in `apostrophe-global` in your project.
+
+Add the following to `layout.html` that all of your pages extend, or to `outerLayout.html` if you have one in `apostrophe-templates/views/` to add configurable Google Analytics and Google site verification.
```
Comment (boutell) [inline README.md]: How about a file they can `{% include %}`? That would be nice for all of the insertions in the layout. They can always copy it in and modify it, or override one or another one at project level, if it's not quite what they want.

Comment (austinstarin) [reply README.md]: Moved this into a view per your suggestion b0ff2f0

**Review by boutell — APPROVED** [review]

**Pull request merged**